examples/FluentUI.Demo.Shared/Pages/Lab/Highlighter/Splitter.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+
```

Comment:
Add above line 4 // Remember to replace the namespace below with your own project's namespace..

Reply:
Done